VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -264,36 +268,40 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
             conference: this.conferenceId,
             participant: p.id,
             pexipParticipant: p.pexipId,
-            current: p.isSpotlighted,
-            new: !p.isSpotlighted
+            current: p.hasSpotlight(),
+            new: !p.hasSpotlight()
         });
-        this.videoCallService.spotlightParticipant(p.pexipId, !p.isSpotlighted, this.conferenceId, p.id);
+        this.videoCallService.spotlightParticipant(p.pexipId, !p.hasSpotlight(), this.conferenceId, p.id);
     }
 
     toggleMuteParticipant(participant: PanelModel) {
         const hearingParticipants = this.participants.filter(x => x.isInHearing());
-        const mutedParticipants = hearingParticipants.filter(x => x.isRemoteMuted);
+        const mutedParticipants = hearingParticipants.filter(x => x.isMicRemoteMuted());
         const p = this.participants.find(x => x.id === participant.id);
 
+        const newMuteStatus = !p.isMicRemoteMuted();
         this.logger.debug(`${this.loggerPrefix} Judge is attempting to toggle mute for participant`, {
             conference: this.conferenceId,
             participant: p.id,
             pexipParticipant: p.pexipId,
-            current: p.isRemoteMuted,
-            new: !p.isRemoteMuted
+            current: p.isMicRemoteMuted(),
+            new: newMuteStatus
         });
-        this.videoCallService.muteParticipant(p.pexipId, !p.isRemoteMuted, this.conferenceId, p.id);
-
-        // check if last person to be unmuted manually
+        this.videoCallService.muteParticipant(p.pexipId, newMuteStatus, this.conferenceId, p.id);
+        if (p instanceof LinkedParticipantPanelModel) {
+            // Participants in a VMR can be muted but they do not return a callback like other participants. Must update manually
+            p.updateParticipant(newMuteStatus, participant.hasHandRaised(), participant.hasSpotlight());
+        }
         if (mutedParticipants.length === 1 && this.isMuteAll) {
+            // check if last person to be unmuted manually
```

Comment:
Didn't we prohibit comments being pushed into master?

Reply:
good spot :)